roles/nameserver/tasks/config.yml

```diff
@@ -10,7 +10,6 @@
     dest: "{{ named_conf_file }}"
     validate: named-checkconf %s
   notify: restart named
-  with_dict: "{{ named_domains }}"
```

Comment:
Discovered this was no longer needed and was needlessly looping over the config.  The j2 template already loops over the dict.